app/src/main/java/com/example/cartoon/MainActivity.kt

```diff
@@ -265,6 +284,22 @@ class MainActivity : ComponentActivity() {
         return TensorImage.fromBitmap(bitmap)
     }
 
+    private fun hideSystemBars() {
+        window.decorView.systemUiVisibility = (
+                View.SYSTEM_UI_FLAG_IMMERSIVE
+                        or View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
```

Comment:
hey @saurabhm13 . This change have to be here only when an app support android 4 and below. I believe whatever you are trying to do here can be done by styling the theme itself. [Reference](https://stackoverflow.com/a/25365193). 
Please try and let us know whether it is possible!

Reply:
Yes, I tried this before, but the result is not as expected. The status bar hides leaving a different background as a black color, and for the navigation bar only the background can be transparent, it will not hide the whole navigation bar.